src/annotator/index.js

```diff
@@ -34,32 +34,49 @@ const appLinkEl = /** @type {Element} */ (
 );
 
 function init() {
```

Comment:
It could be helpful to add some sort of high-level, establishing commenting somewhere in this module. Something like:

```
This code runs in each of the browsing contexts (windows and iframes) involved in annotation:

* It runs in the browsing context that embeds the Hypothes.is client. This is the `host`. It is also a `guest`. In this case, it creates the frame containing the sidebar.
* It runs in any `<iframe enable-annotation />` element (present in the host document?). These are `guest` frames. They establish a connection with the `sidebar` frame created by the `host`.
```

I probably didn't get that quite right. But having something like this could be helpful.

Reply:
Thank you!